napari_clusters_plotter/_utilities.py

```diff
@@ -32,11 +32,12 @@ def get_layer_tabular_data(layer):
     return None
 
 
-def add_column_to_layer_tabular_data(layer, column_name, data):
```

Comment:
@jo-mueller, if we change the arguments in the definition do we additionally need to go through all the other instances where this function is called and should I test that first before merging?

Reply:
Hey @Cryaaa , thanks for the feedback!

Let's say I tested it a few times from Napari and did not run into any issues. But maybe it would make sense to not change the way `add_tabular_data` works but rather creating a new function for this purpose and keep the current function for backwards compatibility - just in case.

Another option: Create the to be added column of data )along with the `NaN`s in the correct position and then write this data to the tabular data with the existing function.